bin/taskcat

```diff
@@ -57,11 +59,25 @@ def main():
                     tcat_instance.set_config("%s/%s" % (taskcat_cfg['global']['qsname'], config_path))
             except Exception as e:
                 print(taskcat.PrintMsg.ERROR + str(e))
+            project_path = '/'.join(tcat_instance.get_config().split('/')[0:-3])
+            project_name = '/'.join(tcat_instance.get_config().split('/')[-3:-2])
+            if "package-lambda" not in taskcat_cfg['global']:
+                taskcat_cfg['global']["package-lambda"] = False
+            if tcat_instance.lambda_build_only and not taskcat_cfg['global']["package-lambda"]:
```

Comment:
I am late in the game :) but many of my questions are already asked and answered in the other's comments.
Quick question here - Where are we setting cat_instance.lambda_build_only ? Did i miss it?

Reply:
in taskcat.yaml (config file)